docs/search/full-text/complex.mdx

```diff
@@ -0,0 +1,270 @@
+---
+title: Advanced Search Queries
+---
+
+## Overview
+
+The `search` function that's generated for your index can accept query objects instead of a query string. Nearly all the query types implemented in Tantivy, our underlying search engine, are exposed to Postgres.
+
+Query objects are composable, allowing for arbitrarily fine-grained queries. A query string itself is parsed into a query object, and can be used anywhere in a query tree with `paradedb.parse`.
+
+`paradedb.parse` is implicitly performed if a query string is passed directly to the `search` function. These three queries are identical:
+
+```sql
+SELECT * FROM search_idx.search('description:shoes');
+
+SELECT * FROM search_idx.search(query => paradedb.parse('description:shoes'));
+
+SELECT * FROM search_idx.search(
+	query => paradedb.term(
+		field => 'description',
+		value => 'shoes'
+	)
+);
+```
+
+## Query Types
+
+The `paradedb` schema exposes the following functions to construct query objects.
+
+### All
+
+This query type indiscriminately matches every document in the index, assigning a uniform score of 1.0 to each.
+
+```sql
+SELECT * FROM search_index.search(
```

Comment:
nit: Let's keep the name of our example index consistent across the documentation. It was frustrating for me to copy/paste this code and then have to change the name of the example index. It would be nice to call it `search_idx` since that's the one created in the quickstart.

Reply:
Yup that's a copy paste error thanks for catching.